resources/views/event/review/create.blade.php

```diff
@@ -1,9 +1,32 @@
 @extends('layouts.app')
 
-@section('content')
-    <h1> create organization review on {{ $event->name }}</h1>
-    {!! Form::open(['url' => 'event/'.$event->id.'/review']) !!}
-        @include('event.review.partials.form', ['submitButtonText' => 'Add Review!'])
-    {!! Form::close() !!}
-    
+        @section('content')
```

Comment:
You should always check the changed files. If you didn't make these changes, then **don't leave them and restore everything as it was**